crates/utils/re_auth/src/oauth/api.rs

```diff
@@ -2,12 +2,17 @@
 
 use std::sync::LazyLock;
 
-use super::{RefreshToken, User};
+use base64::{Engine as _, prelude::BASE64_URL_SAFE_NO_PAD};
+use sha2::{Digest as _, Sha256};
 
-static API_BASE_URL: LazyLock<String> = LazyLock::new(|| {
-    std::env::var("RERUN_AUTH_API_BASE_URL")
+use crate::oauth::OAUTH_CLIENT_ID;
+
+use super::RefreshToken;
+
+static WORKOS_API: LazyLock<String> = LazyLock::new(|| {
+    std::env::var("WORKOS_API_BASE_URL")
```

Comment:
Since it's standard OAuth2 at this point, maybe the env var could be called something not vendor specific. I like the `RERUN_OAUTH_CLIENT_ID` name you picked, so maybe `RERUN_OAUTH_SERVER_URL`?